src/audio/Kconfig

```diff
@@ -140,7 +140,9 @@ config COMP_IIR
 	bool "IIR component"
 	select COMP_BLOB
 	default y
+	depends on COMP_MODULE_ADAPTER
 	select MATH_IIR_DF1
+
```

Comment:
this empty line is quite unusual, don't remember seeing it before - before "help" in Kconfig

Reply:
Oops, fixed.